motor/motor_asyncio.py

```diff
@@ -18,7 +18,7 @@
 from .frameworks import asyncio as asyncio_framework
 from .metaprogramming import create_class_with_framework
 
-__all__ = ['AsyncIOMotorClient']
+__all__ = ['AsyncIOMotorClient, AsyncIOMotorClientEncryption']
```

Comment:
`['AsyncIOMotorClient, Asy...` -> `['AsyncIOMotorClient', 'Asy...`

Reply:
Oops not sure how that got there, fixed!